cmd/maya-apiserver/app/server/volume_endpoint_v1alpha1.go

```diff
@@ -35,20 +33,20 @@ func (s *HTTPServer) volumeV1alpha1SpecificRequest(resp http.ResponseWriter, req
 	case "PUT", "POST":
```

Comment:
Again, IMO only POST verb should be associated for a resource creation.
PUT should be used while updating a resource.

Reply:
ok